lib/structuraid_core/designers/footings/steps/centric_isolated/set_reinforcement_layers_coordinates_to_footing.rb

```diff
@@ -0,0 +1,101 @@
+require 'interactor'
+require 'matrix'
+
+module StructuraidCore
+  module Designers
+    module Footings
+      module Steps
+        module CentricIsolated
+          # Runs the structural bending analysis and design for a centric isolated footing, generating the required reinforcement ratio and checking that the provided one is enough
+          class SetReinforcementLayersCoordinatesToFooting
```

Comment:
I didn't get too much of this code, but it doesn't really feel like a design step, but rather an artifact of something that needs to be done on our system after the reinforcement is generated. Can we do it in a different place? Let's discuss this